partitura/musicanalysis/note_features.py

```diff
@@ -72,9 +73,11 @@ def list_note_feats_functions():
 
 
 def make_note_features(
-    part: Union[score.Part, score.PartGroup, List],
+    part: ScoreLike,
     feature_functions: Union[List, str],
     add_idx: bool = False,
+    include_empty_features: bool = True,
```

Comment:
the function now breaks if `include_empty_features` is False but no features can be found. Is this intended?

Reply:
Do you have a test file? So I can test while fixing it?